Cargo.lock

```diff
@@ -2418,7 +2418,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4979f22fdb869068da03c9f7528f8297c6fd2606bc3a4affe42e6a823fdb8da4"
 dependencies = [
  "cfg-if",
- "windows-targets 0.52.6",
+ "windows-targets 0.48.5",
```

Comment:
Why `windows-targets` get downgraded?

Reply:
Hm, it seems `cargo update -p rustix` caused that - I've reverted it manually now.